src/lib/y2partitioner/actions.rb

```diff
@@ -21,22 +19,10 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
-require "y2storage"
-
 module Y2Partitioner
-  module Refinements
-    # Temporary refinement for Y2Storage::Filesystems::Type. It should be
-    # replaced by a proper class or any other mechanism at some point.
-    module FilesystemType
-      refine Y2Storage::Filesystems::Type do
-        def formattable?
-          [:btrfs, :ext2, :ext3, :ext4, :swap, :vfat, :xfs].include?(to_sym)
-        end
-
-        def encryptable?
-          formattable?
-        end
-      end
-    end
+  # Namespace to group all the actions of the expert partitioner
+  module Actions
   end
 end
+
+require "y2partitioner/actions/delete_blk_device"
```

Comment:
Looks like the comment I added here before disappeared 🤔  I will repeat it:

This `y2partitioner/actions.rb` seems to be there to make possible to require all the actions at once, but it only requires one of the existing actions. It's a little bit confusing to me. I would either kill the file or make sure it actually requires all the actions. Having a file there that just requires another random file looks weird.

BTW, it's fun how git detects this as a renaming of the old refinement. :-)

Reply:
I forgot to include the rest of actions. I like having a separated file for the module definition.